src/layout/Number/config.ts

```diff
@@ -0,0 +1,27 @@
+import { CG } from 'src/codegen/CG';
+import { ExprVal } from 'src/features/expressions/types';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Presentation,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: true,
+    renderInButtonGroup: false,
+    renderInAccordion: true,
+    renderInAccordionGroup: false,
+    renderInCards: true,
+    renderInCardsMedia: false,
+  },
+})
+  .addTextResource(
+    new CG.trb({
+      name: 'title',
+      title: 'Title',
+      description: 'The title of the value',
+    }),
+  )
```

Comment:
I think this should just set `rendersWithLabel: true` above, and then you can add `.extendTextResources(CG.common('TRBLabel'))` instead of a manual `title` text resource binding. That way this component renders with the built-in label (which also supports description and helpText), and in those cases where you need to skip rendering the label (such as in a panel listing), you can render it with forced 'direct rendering' to avoid the label:

```JSX
<GenericComponent
  node={someNumberNode}
  overrideDisplay={{
    directRender: true,
  }}
/>
```